testslide/strict_mock.py

```diff
@@ -634,7 +634,11 @@ def __validate_attribute_type(self, name: str, value: Any) -> None:
             return
 
         if self._template is not None:
-            annotations = get_type_hints(self._template)
+            try:
+                annotations = get_type_hints(self._template)
+            except KeyError:
```

Comment:
can we do except Exception just to make sure we dont break tests when we fail to get typehints for any funny modules?

Reply:
KeyError seemed more specific to the modules with different `__module__` . Are there any other failures that expose a bug with get_type_hints? I couldn't see the test failures since it's private.

Reply:
https://www.python.org/dev/peps/pep-0563/#runtime-annotation-resolution-and-class-decorators it can throw NameError on metaclasses